formidable_color_picker/serializers.py

```diff
@@ -9,7 +8,6 @@
 class ColorPickerFieldSerializer(FieldSerializer):
 
     type_id = 'color_picker'
-    parameters = serializers.JSONField()
```

Comment:
To be honest, I've not checked but it seems that this value is required(But maybe with a ```write_only=True``` argument ). Am I wrong? 

```python
parameters = serializers.JSONField(write_only=True)
```

Reply:
it didn't complain when I removed it in fact. I've tested locally using my copy of django-formidable with this branch.